WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -320,6 +320,11 @@ private void OnCoinJoinPhaseChanged(CoinJoinProgressEventArgs coinJoinProgress)
 	{
 		switch (coinJoinProgress)
 		{
+			case RoundEndedUserPressedPause:
+				StopCountDown();
+				_stateMachine.Fire(Trigger.WalletStoppedCoinJoin);
```

Comment:
I don't think it is valid to touch the state machine here. The state machine is controlled in StatusChanged. OnCoinJoinPhaseChanged only updating the UI display. I would display Stopping the CoinJoin... or something similar.

Reply:
I agree this is not really clean.
I don't think the UX will be great if we create another message for when the CJ is stopping as you suggested. because in some cases this message will appear just few hundreds of a s so the user might just see a flickering without having time to read the message.
<details>
<summary>See by yourself:</summary>

![WalletWasabi Fluent Desktop_J7FFXM1XLI](https://user-images.githubusercontent.com/16029533/186716960-d2ab4106-c4f9-47bb-9447-2dd3e3125d96.gif)
</details>

There are basically 2 solutions:
1. DIsplay the message "Coinjoin is paused" directly. I originally discarded this solution as it would display for some ms that CoinJoin is paused but with the pause button still visible
2. Do nothing, this introduces some kind of a lag as you can see on my GIF in the original PR message.

Reply:
There is a way how music box is working. The user makes some actions on the music box control, the command goes into CoinJoinManager and that will emit an event which will change the music box appearance. In this way what the user can see is always the real state of the business logic. This means it is not good if the music box is directly changed by the buttons. I wrote this just for clarification. 

Because of this IMO the two states cannot be avoided, however, there might be a trick. What if you do not touch anything in the in/between state - maybe that is what you refer to, do nothing. So there won't be any flickering but just a short delay after pressing the pause button. I think that would be acceptable. Or do not touch the messages but disable the pause button in the in-between state.

Reply:
yes I was referring to this. At first I made it like that and I should revert to this as it's the best. Furthermore, as the timer is directly it looks nice as little loading. I will push soon and mark as resolved. Thank you for the review!